{"uid":"bded6c6930f1116a","name":"test_precision[Test for Subsocial]","fullName":"tests.test_network_parameters.TestPrecision#test_precision","historyId":"f90aab6720febe770894df7b45dabf90","time":{"start":1667521294573,"stop":1667521296225,"duration":1652},"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","statusTrace":"self = <tests.test_network_parameters.TestPrecision object at 0x7f609d0f1fd0>\nchain = <tests.utils.chain_model.Chain object at 0x7f609d61bc40>\n\n    def test_precision(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7f609d61bc40>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] Subsocial\n\ntests/utils/chain_model.py:29: TimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","statusTrace":"self = <tests.test_network_parameters.TestPrecision object at 0x7f609d0f1fd0>\nchain = <tests.utils.chain_model.Chain object at 0x7f609d61bc40>\n\n    def test_precision(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7f609d61bc40>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] Subsocial\n\ntests/utils/chain_model.py:29: TimeoutError","steps":[],"attachments":[{"uid":"5708afac017fc1ff","name":"stdout","source":"5708afac017fc1ff.txt","type":"text/plain","size":54}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_network_parameters"},{"name":"subSuite","value":"TestPrecision"},{"name":"host","value":"fv-az119-190"},{"name":"thread","value":"1630-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_network_parameters"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"chain","value":"<tests.utils.chain_model.Chain object at 0x7f609d61bc40>"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"a682f69a8fb178cc","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667521282806,"stop":1667521284556,"duration":1750}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":137,"skipped":0,"passed":0,"unknown":0,"total":137},"items":[{"uid":"157c8dbb9024032b","reportUrl":"https://nova-wallet.github.io/test-runner/1833//#testresult/157c8dbb9024032b","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667506158448,"stop":1667506160602,"duration":2154}},{"uid":"fb4529414595b0e6","reportUrl":"https://nova-wallet.github.io/test-runner/1832//#testresult/fb4529414595b0e6","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667491858193,"stop":1667491859887,"duration":1694}},{"uid":"8211a6782238a8ba","reportUrl":"https://nova-wallet.github.io/test-runner/1831//#testresult/8211a6782238a8ba","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667477431187,"stop":1667477433277,"duration":2090}},{"uid":"5bf34e7a0ef022f8","reportUrl":"https://nova-wallet.github.io/test-runner/1830//#testresult/5bf34e7a0ef022f8","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667462964057,"stop":1667462965913,"duration":1856}},{"uid":"3f8124a801655298","reportUrl":"https://nova-wallet.github.io/test-runner/1829//#testresult/3f8124a801655298","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667462180668,"stop":1667462182617,"duration":1949}},{"uid":"4c7ef2862f7ee00e","reportUrl":"https://nova-wallet.github.io/test-runner/1810//#testresult/4c7ef2862f7ee00e","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667192068634,"stop":1667192070821,"duration":2187}},{"uid":"c2230c44c53dcbaf","reportUrl":"https://nova-wallet.github.io/test-runner/1809//#testresult/c2230c44c53dcbaf","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667181252259,"stop":1667181254180,"duration":1921}},{"uid":"9dab6ced03e28950","reportUrl":"https://nova-wallet.github.io/test-runner/1808//#testresult/9dab6ced03e28950","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667161353735,"stop":1667161356036,"duration":2301}},{"uid":"6dc9c2bb70ac7330","reportUrl":"https://nova-wallet.github.io/test-runner/1807//#testresult/6dc9c2bb70ac7330","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667147233503,"stop":1667147235633,"duration":2130}},{"uid":"41d0a2a746c0a185","reportUrl":"https://nova-wallet.github.io/test-runner/1806//#testresult/41d0a2a746c0a185","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667133345474,"stop":1667133347026,"duration":1552}},{"uid":"1ef2719b32ad6920","reportUrl":"https://nova-wallet.github.io/test-runner/1805//#testresult/1ef2719b32ad6920","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667118335875,"stop":1667118338152,"duration":2277}},{"uid":"41abd529d6ae6ef2","reportUrl":"https://nova-wallet.github.io/test-runner/1804//#testresult/41abd529d6ae6ef2","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667105347206,"stop":1667105348900,"duration":1694}},{"uid":"f1b827f42af8f33c","reportUrl":"https://nova-wallet.github.io/test-runner/1803//#testresult/f1b827f42af8f33c","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667094804787,"stop":1667094807054,"duration":2267}},{"uid":"1fd49256668000a5","reportUrl":"https://nova-wallet.github.io/test-runner/1802//#testresult/1fd49256668000a5","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667074878182,"stop":1667074879843,"duration":1661}},{"uid":"191df5c61f7a1c59","reportUrl":"https://nova-wallet.github.io/test-runner/1801//#testresult/191df5c61f7a1c59","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667060794862,"stop":1667060796043,"duration":1181}},{"uid":"83d505eca7b6c052","reportUrl":"https://nova-wallet.github.io/test-runner/1800//#testresult/83d505eca7b6c052","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667046847929,"stop":1667046849452,"duration":1523}},{"uid":"22ce108bedfcbcae","reportUrl":"https://nova-wallet.github.io/test-runner/1799//#testresult/22ce108bedfcbcae","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667032003054,"stop":1667032005395,"duration":2341}},{"uid":"eda0101273901d46","reportUrl":"https://nova-wallet.github.io/test-runner/1798//#testresult/eda0101273901d46","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667018275747,"stop":1667018277398,"duration":1651}},{"uid":"4475922a2dcdd7c","reportUrl":"https://nova-wallet.github.io/test-runner/1797//#testresult/4475922a2dcdd7c","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1667006842835,"stop":1667006844826,"duration":1991}},{"uid":"95ef1b04a5f075b2","reportUrl":"https://nova-wallet.github.io/test-runner/1796//#testresult/95ef1b04a5f075b2","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666988537801,"stop":1666988539204,"duration":1403}}]},"tags":[]},"source":"bded6c6930f1116a.json","parameterValues":["<tests.utils.chain_model.Chain object at 0x7f609d61bc40>"]}